### DISCUSSION TITLE
RustDesk cannot interact with Little Snitch network pop-up

### DISCUSSION THREAD

**Discussion by rekabis** [rustdesk/rustdesk#5794] [General]
### Bug Description

The latest RustDesk cannot interact with a pop-up from the Little Snitch network firewall.

![image](https://github.com/rustdesk/rustdesk/assets/7185825/53d0e95d-608a-4cfc-a0a6-c8737b4181c8)

I suspect this is because the Little Snitch network firewall pop-up does not have any entry in the “active applications” of the system (the kill window to the left), and cannot be reached via a command-tab to move through the open windows.

### How to Reproduce

- Connect to the Mac with RustDesk
- Install Little Snitch onto a Mac.
- Set the Firewall component to always ask for connection permission of any app
- Open up an unsigned app, such as LibreWolf, and try to connect to any Internet or network resource.
- The window in the screenshot in the bug description will open up, requesting permission to access said Internet resource.

This pop-up window will not be clickable remotely, and cannot be dismissable without disabling/restarting the Little Snitch network filter, blocking most of the screen.

### Expected Behavior

To be able to interact with the remote pop-up so it can be processed, allowing LibreWolf to connect to the router’s Admin interface.

### Operating system(s) on local side and remote side

Windows 10 -> MacOS Ventura 13.5.2

### RustDesk Version(s) on local side and remote side

1.1.9 -> 1.1.8

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/7185825/4042d947-093e-4f77-b820-eceeccf6e2fc)

**Comment by dazey77** [comment]
I have this problem too!

**Reply by shaman1010** [reply]
In the settings Little Snitch allow access to the CLI.
